src/airdrop/src/lib.rs

```diff
@@ -196,7 +198,7 @@ pub fn generate_code() -> CustomResult<CodeInfo> {
         let principal_state = authorized.get_mut(&caller_principal).unwrap();
 
         // generate a new code
-        let code = get_pre_codes()?;
+        let code = utils::get_pre_codes()?;
```

Comment:
the prefix `utils::` is useless here as `get_pre_codes` is imported at the top of the module